buildSrc/src/main/kotlin/android-sample-convention.gradle.kts

```diff
@@ -44,17 +42,6 @@ android {
             signingConfig = signingConfigs.getByName("debug")
         }
     }
-
-    compileOptions {
-        sourceCompatibility = JavaVersion.VERSION_1_8
-        targetCompatibility = JavaVersion.VERSION_1_8
-    }
-    kotlinOptions {
-        jvmTarget = "1.8"
```

Comment:
I believe these are still necessary, no?

Reply:
Android gradle plugin (v4 and above) uses `compileOptions` with java 8 by default. I will try find references...

The default target version for Kotlin/JVM compilations is now 1.8
https://kotlinlang.org/docs/whatsnew15.html#new-default-jvm-target-1-8